pyproject.toml

```diff
@@ -155,12 +154,20 @@ include = '''
 '''
 exclude = ''
 
+[project]
+requires-python = ">=3.9,<=3.12"
```

Comment:
Wouldn't this reject `3.12.1`?  Theoretically...  We have a similar thing above (`python = ">=3.9, <3.13"`) so perhaps they should match?  Maybe poetry can respect this entry here and we can avoid duplication?

Reply:
Duplicated until poetry can be made to read from this.